core/app/models/spree/stock_transfer.rb

```diff
@@ -100,6 +104,18 @@ def close(closed_by)
 
     private
 
+    def ensure_transfer_items_valid
+      invalid_items = transfer_items.select(&:invalid?)
+
+      if invalid_items.blank?
```

Comment:
I don't like that as errors could be non-empty from another validation/operation and I don't want this to return a false-positive